neurostore-openapi.yml

```diff
@@ -133,10 +135,7 @@ paths:
         content:
           application/json:
             schema:
-              oneOf:
-                - properties: {}
-                  nullable: true
-                - $ref: '#/components/schemas/study'
+              $ref: '#/components/schemas/study'
```

Comment:
keep an eye on this to see if we can represent a nullable reference _and_ generate a typescript library.